content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -65,7 +65,10 @@ Then you can explore the API with curl, wget, or a browser, replacing localhost
 with [::1] for IPv6, like so:
 
 ```shell
-$ curl http://localhost:8080/api/
+curl http://localhost:8080/api/
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output